test/ngAnimate/integrationSpec.js

```diff
@@ -319,6 +319,40 @@ describe('ngAnimate integration tests', function() {
         }
       });
     });
+
+    it('should trigger callbacks at the start and end of an animation',
+      inject(function($rootScope, $rootElement, $animate, $compile) {
+
+      ss.addRule('.animate-me', 'transition:2s linear all;');
+
+      var parent = jqLite('<div></div>');
+      html(parent);
+
+      var element = jqLite('<div ng-if="exp" class="animate-me"></div>');
+      parent.append(element);
```

Comment:
Given that you `find` this element again later on in the test, you might as well tidy up setup here to something like:

```
var element,
    parent = jqLite('<div><div ng-if="exp" class="animate-me"></div></div>');
html(parent);
```